libcloud/compute/drivers/gce.py

```
@@ -1854,6 +1885,65 @@ def ex_destroy_address(self, address):
         self.connection.async_request(request, method='DELETE')
         return True
 
+    def ex_delete_image(self, image):
+        """
+        Delete a specific image resource.
+
+        :param  image: Image object to delete
+        :type   image: ``str`` or :class:`GCENodeImage`
+
+        :return: True if successfull
+        :rtype:  ``bool``
+        """
+        if not hasattr(image, 'name'):
+            image = self.ex_get_image(image)
+
+        request = '/global/images/%s' % (image.name)
+        self.connection.async_request(request, method='DELETE')
+        return True
+
+    def ex_deprecate_image(self, image, replacement, state=None):
+        """
+        Deprecate a specific image resource.
+
+        :param  image: Image object to deprecate
+        :type   image: ``str`` or :class: `GCENodeImage`
+
+        :param  replacement: Image object to use as a replacement
+        :type   replacement: ``str`` or :class: `GCENodeImage`
+
+        :param  state: State of the image
+        :type   state: ``str``
+
+        :return: True if successfull
+        :rtype:  ``bool``
+        """
+        if not hasattr(image, 'name'):
+            image = self.ex_get_image(image)
+
+        if not hasattr(replacement, 'name'):
+            replacement = self.ex_get_image(replacement)
+
+        if state is None:
+            state = 'DEPRECATED'
+
+        possible_states = ['DELETED', 'DEPRECATED', 'OBSOLETE']
+
+        if state not in possible_states:
```

Should throw `ValueError` instead of returning `False`